data/default_must_gather_versions.json

```diff
@@ -1,17 +1,22 @@
 {
 	"4.7": {
 		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v2.6.5",
-		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8",
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8:v4.7",
+		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v4.7"
 	},
 	"4.8": {
 		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v2.6.5",
-		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8",
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8:v4.8",
+		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v4.8"
 	},
 	"4.9": {
-		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v2.6.5",
-		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8",
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v4.9.3",
+		"odf": "registry.redhat.io/odf4/ocs-must-gather-rhel8:v4.9",
```

Comment:
based on what @slaviered wrote - should it be ocs or odf?

Reply:
based on what @slaviered wrote, it should be ocs. I have updated the PR